.ci/publish_linux.sh

```diff
@@ -1,13 +1,6 @@
 #!/usr/bin/env bash
 set -e
 
-if [ "${TRAVIS_PULL_REQUEST}" = "false" ]; then
```

Comment:
Can you explain why its OK to delete this check? It seems like we would still want the linux publish script to not publish based on builds for pull requests.

Reply:
Because `deploy` step in `travis.yml` is set to `on.tags: true`. This means that this step will be executed only on a tag push, not during a PR, or even a normal push on `master`. So the condition in the bash script is redundant.

You can see effective execution of the pipeline to a push to `master` on my fork here: https://travis-ci.org/letsencrypt/pebble/builds/498856123

You will see that the deploy step is not executed, with an explicit message about that.

Reply:
That makes sense! Thanks :+1: